doc/application-api-guide/api_guide_lines.dox

```diff
@@ -7,7 +7,7 @@
 
 /**
 
-@page api_guide_lines  API Guide Lines
+@page api_guide_lines  API Developer Guide Lines
```

Comment:
I don't know if this is relevant to this PR, but "guidelines" is one word. There's at least one other occurrence (in application-api-guide).